prepare_data.sh

```diff
@@ -6,17 +6,28 @@ INPUT_SHP_ARCHIVE=${DATASET_PREFIX}_2018-02-16.tgz
 INPUT_SHP_FILE=export/routegeometry.shp
 OUTPUT_MBTILES=${DATASET_PREFIX}_LV95_WM.mbtiles
 
+POI_DATASET_PREFIX=data/station-didok
+
 echo "Convert into GEOJSON LV95 from LV03 shapefile archive..."
 ogr2ogr -f "GeoJSON" ${DATASET_PREFIX}_LV95.geojson /vsitar/${INPUT_SHP_ARCHIVE}/${INPUT_SHP_FILE} -s_srs EPSG:21781 -t_srs EPSG:2056 -select land,route_id,surface
 
 echo "Run python script to convert from LV95 to 'Web Swiss' (fake web mercator)..."
 python3 lv95_web_swiss/lv95_web_swiss.py ${DATASET_PREFIX}_LV95.geojson ${DATASET_PREFIX}_web_swiss_wrong_crs.geojson
 
+echo "Converting POI SBB to projection LV95..."
+ogr2ogr -f "GeoJSON" -s_srs EPSG:4326 -t_srs EPSG:2056 ${POI_DATASET_PREFIX}-lv95.geojson
```

Comment:
I think this is missing the input file name. And you should probably also document how to obtain the input file ;-)

Reply:
fixed in commit bc5e5bb